engine/src/main/java/org/terasology/rendering/dag/RenderGraph.java

```diff
@@ -166,4 +170,27 @@ public void dispose() {
         }
         nodeMap.clear();
     }
+
+    /*private void matchDependencyConnections(Node fromNode, Node toNode){
+        /**TODO REMOVE THIS IF WE MAKE THIS CHANGE FOR EVERY NODE
+         *!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!*//*
+        if((fromNode instanceof NewAbstractNode)&&(toNode instanceof NewAbstractNode)){
+            NewAbstractNode fromN=(NewAbstractNode) fromNode;
+            NewAbstractNode toN=(NewAbstractNode) toNode;
+
+            List<Name> fromOutputs = fromN.getOutputConnections();
```

Comment:
Please, use toNode and fromNode. We are not working in the 80s anymore, where we have 8 length variable name limits.

Reply:
This code is probably not going to be used anytime soon, but...help me come up with names then for future reference
`NewAbstractNode fromN=(NewAbstractNode) fromNode;  `
both are Node. Or you could just cast without saving it to a variable